keyboards/handwired/lovelive9/lovelive9.h

```diff
@@ -1,20 +1,10 @@
 #pragma once
 
 #include "quantum.h"
-#ifdef RGBLIGHT_ENABLE
-//rgb led driver
-#endif
-#define LAYOUT( \
+#define LAYOUT_ortho_1x9( \
```

Comment:
As there's only one physical layout supported, in my opinion this should just be `LAYOUT`, and then the alias definition on Line 10 should be deleted.